index.html

```diff
@@ -3,14 +3,36 @@
 
 <head>
     <title>Project UFEM</title>
+    <link rel="stylesheet" href="./css/styles.css">
 </head>
 
 <body>
     <div class="banner">
         sth is gonna be there
     </div>
     <header>
-        <h1>Hello world!</h1>
+        <nav class="google-navigation">
+            <ul>
+                <li class="google-navigation__item">
+                    <a href="#" class="google-navigation__link">Gmail</a>
+                </li>
+                <li class="google-navigation__item">
+                    <a href="#" class="google-navigation__link">Images</a>
+                </li>
+                <li class="google-navigation__item">
+                    <button type="button" class="google-navigation__button products-button">
+                        <svg viewBox="0 0 24 24" width="30px" height="30px">
```

Comment:
It would be way better if we set the size of an icon using _CSS_ instead of `width` and `height` attributes. We can set the class for `<svg>...</svg>` tag.
```suggestion
                        <svg class="icon main-nav__icon" viewBox="0 0 24 24">
```
This way we can define core styling for `.icon` and then set/change it using `.main-nav__icon` _"element"_ or `icon--<MODIFIER_NAME>` _"modifier"_

You can read more about _block__element--modifier_ concept at [BEM - Block Element Modifier](http://getbem.com/) page

Reply:
done